tests/test_pages.py

```diff
@@ -20,7 +20,7 @@ class Item:
     name: str
 
 
-def test_page_object():
+def test_page_object() -> None:
```

Comment:
By the way, I think there is nothing wrong with this opt-in approach using `-> None` annotations, it looks fine.

Reply:
Yeah, it would seem so. The remaining challenge here is to ensure that all of our tests has this annotation. One of the advantage of the `--check-untyped-defs` that I see is that it's able to catch any unannotated code or tests.